test/libsolidity/SemanticTest.cpp

```diff
@@ -131,6 +132,19 @@ void SemanticTest::initializeBuiltins()
 	{
 		return util::toBigEndian(u256(0x1234));
 	};
+	soltestAssert(m_builtins.count("accountBalance") == 0, "");
+	m_builtins["accountBalance"] = [this](FunctionCall const& _call) -> std::optional<bytes>
```

Comment:
Can we only use a single builtin called `balance` which has an optional parameter?

Reply:
Yes, thats possible.